test/plugin/test_out_google_cloud.rb

```diff
@@ -346,6 +346,14 @@ def test_statusz_endpoint
     end
   end
 
+  def test_monitoring_config
+    setup_gce_metadata_stubs
+    create_driver(%(
```

Comment:
What is this testing? That the code doesn't crash? Can you please add a comment to that effect?